src/diffusers/models/attention.py

```diff
@@ -175,7 +175,7 @@ def __init__(
             self.norm_out = nn.LayerNorm(inner_dim)
             self.out = nn.Linear(inner_dim, self.num_vector_embeds - 1)
 
-    def forward(self, hidden_states, encoder_hidden_states=None, timestep=None, return_dict: bool = True):
+    def forward(self, hidden_states, encoder_hidden_states=None, timestep=None, mask=None, return_dict: bool = True):
```

Comment:
```suggestion
    def forward(self, hidden_states, encoder_hidden_states=None, timestep=None, attention_mask=None, return_dict: bool = True):
```

Reply:
Let's try to match the naming 1-to-1 here with transformers: https://github.com/huggingface/transformers/blob/7032e0203262ebb2ebf55da8d2e01f873973e835/src/transformers/models/bert/modeling_bert.py#L278

Reply:
done here!
https://github.com/huggingface/diffusers/pull/1428/commits/2b30fbc7d384335f7aef047695b502dddb01cf16